README.md

```diff
@@ -26,6 +26,14 @@ To install `snarkjs`, just run:
 npm install -g snarkjs@latest
 ```
 
+### Download SRS File
+
+To create a ZKP you would need to download the correct SRS file (based on your circuit size). For example, if your circuit has around 65K constraints then you would need to download the following file https://storage.googleapis.com/zkevm/ptau/powersOfTau28_hez_final_16.ptau.
```

Comment:
sorry for the slow review, we should add a link to https://github.com/iden3/snarkjs#7-prepare-phase-2 which is more user friendly

Reply:
also it'd be good to add a `wget` premade command for the lazy asses